docs/sdk/authentication/session-sigs/get-session-sigs.md

```diff
@@ -1,113 +1,331 @@
+
+The available Lit Resource options are:
+
+##### `LitAccessControlConditionResource`
+
+Imported from the `@lit-protocol/auth-helpers` package:
+
+```ts
+import { LitAccessControlConditionResource } from "@lit-protocol/auth-helpers";
+```
+
+This resource specifies a Lit Access Control Conditions (ACC) the `ability` is being granted for. Specifying `"*"` grants the `ability` to any ACC, regardless of what the conditions are. To grant the `ability` for a specific ACC, you can specify the hashed key value of the ACC in place of `"*"` like so:
+
+<!-- TODO How to get the hashed key value of the ACC -->
+
+```ts
+new LitAccessControlConditionResource("")
+```
+
+##### `LitPKPResource`
+
+Imported from the `@lit-protocol/auth-helpers` package:
+
+```ts
+import { LitPKPResource } from "@lit-protocol/auth-helpers";
+```
+
+This resource specifies what Lit Programmable Key Pair (PKP) the `ability` is being granted for. Specifying `"*"` grants the `ability` for all PKPs. To grant the `ability` for a specific PKP, you can specify the token ID of the PKP in place of `"*"` like so:
+
+```ts
+new LitPKPResource("42")
+```
+
+##### `LitRLIResource`
+
+Imported from the `@lit-protocol/auth-helpers` package:
+
+```ts
+import { LitRLIResource } from "@lit-protocol/auth-helpers";
+```
+
+This resource specifies what Lit Capacity Credit the `ability` is being granted for. Specifying `"*"` grants the `ability` for all Capacity Credits. To grant the `ability` for a specific Capacity Credit, you can specify the token ID in place of `"*"` like so:
+
+```ts
+new LitRLIResource("42")
+```
+
+##### `LitActionResource`
+
+Imported from the `@lit-protocol/auth-helpers` package:
+
+```ts
+import { LitActionResource } from "@lit-protocol/auth-helpers";
+```
+
+This resource specifies what Lit Actions the `ability` is being granted for. Specifying `"*"` grants the `ability` for all Lit Actions, regardless of who created them. To grant the `ability` for a specific Lit Action, you can specify an IPFS Content Identifier (CID) in place of `"*"` like so:
+
+```ts
+new LitActionResource("QmX8bs6vhiFvwgYS39ZqDW421D1885n3KNzqqwRhTPTP8e")
+```
+
+#### `ability`
+
+`ability` is the second part of each Lit Resource Request, and it specifies what we're allowing the Session Keys to do with a Lit Resource.
+
+Each available ability can be specified using the `LitAbility` `enum` which can be imported from the `@lit-protocol/auth-helpers` package:
+
+```ts
+import { LitAbility } from "@lit-protocol/auth-helpers";
+```
+
+The abilities available to grant are:
+
+- `AccessControlConditionDecryption` This grants the ability to decrypt some Lit encrypted data if the Access Control Conditions are satisfied
+<!-- TODO Is this correct? -->
+- `AccessControlConditionSigning` This grants the ability to sign data using threshold signing by the Lit nodes in the network if the Access Control Conditions is satisfied.
```

Comment:
@joshLong145 I'm not sure if this statement is correct

Reply:
@DashKash54 @hwrdtm can you confirm this?

Reply:
I think this was used for `jwt` features hence `signing` being referred to but we should get confirmation